Recolor.php

```diff
@@ -56,4 +56,14 @@ public function image($imageName, $newFilename)
         imagepng($image, $this->imageFolder . $newFilename);
         imagedestroy($image);
     }
-}
\ No newline at end of file
+}
+
+/**
+ * Let's actually run the code!
+ */
+
+// Instanciate a new instance of the class
+$recolor = new Recolor();
```

Comment:
We'll create a new instance of the class, with the `new` keyword. This is called instantiating the class. We assign it to a variable, to hold the instance. We can now worth with that instance.